docs/getting-started/zowe-security-overview.md

```diff
@@ -29,8 +29,8 @@ A certificate contains an identity (a hostname, or an organization, or an indivi
 A certificate can be self-signed or issued by a Certificate Authority (CA). A CA is a trusted organization which provides infrastructure for creation, validation and revocation of the certificates according to the contemporary security standards.
 
 **Note:** 
-For testing purposes of Zowe, it is acceptable to use certificates issued and signed either by the company's local CA, or even self-signed certificates issued by Zowe security tools specific for the target technology platform.
-Use of self-signed certificates, however, is not recommended for production environments.
+For testing purposes of Zowe, it is acceptable to use certificates issued and signed either by the company's local CA, or even certificates issued by Zowe security tools and signed by generated CA specific for the target technology platform.
```

Comment:
- Please use the Docusaurus syntax for admonitions. See previous statement for the URL.
- We should spell out "CA" on first use.